LayoutTests/inspector/css/forcePseudoState.html

```diff
@@ -0,0 +1,92 @@
+        {forceablePseudoClass: WI.CSSManager.ForceablePseudoClass.FocusWithin, expectedBackgroundColor: "rgb(0, 0, 40)"},
+        {forceablePseudoClass: WI.CSSManager.ForceablePseudoClass.Hover, expectedBackgroundColor: "rgb(0, 0, 50)"},
+        {forceablePseudoClass: WI.CSSManager.ForceablePseudoClass.Visited, expectedBackgroundColor: "rgb(0, 0, 60)"},
+    ].forEach(({forceablePseudoClass, expectedBackgroundColor}) => {
+        suite.addTestCase({
+            name: "CSS.forcePseudoState." + forceablePseudoClass,
+            async test() {
+                await nodeStyles.refreshIfNeeded();
+
+                InspectorTest.assert(Object.shallowEqual(targetNode.enabledPseudoClasses, []), "Should not have any enabled pseudo classes.");
+                InspectorTest.assert(nodeStyles.computedStyle.propertyForName("background-color").value === "rgb(0, 0, 0)", "Should have expected initial style.");
+
+                InspectorTest.log("Forcing pseudo class...");
+                await Promise.all([
+                    targetNode.awaitEvent(WI.DOMNode.Event.EnabledPseudoClassesChanged),
+                    targetNode.setPseudoClassEnabled(forceablePseudoClass, true),
+                ]);
+                await nodeStyles.refreshIfNeeded();
+                InspectorTest.expectShallowEqual(targetNode.enabledPseudoClasses, [forceablePseudoClass], "Should have one enabled pseudo class.");
+                InspectorTest.expectEqual(nodeStyles.computedStyle.propertyForName("background-color").value, expectedBackgroundColor, "Should change styles.");
+
+                InspectorTest.log("Removing forced pseudo class...");
+                await Promise.all([
+                    targetNode.awaitEvent(WI.DOMNode.Event.EnabledPseudoClassesChanged),
+                    targetNode.setPseudoClassEnabled(forceablePseudoClass, false),
+                ]);
+                await nodeStyles.refreshIfNeeded();
+                InspectorTest.expectShallowEqual(targetNode.enabledPseudoClasses, [], "Should not have any enabled pseudo classes.");
+                InspectorTest.expectEqual(nodeStyles.computedStyle.propertyForName("background-color").value, "rgb(0, 0, 0)", "Should change back to initial style.");
+            },
+        });
+    });
+
+    WI.domManager.requestDocument((documentNode) => {
+        documentNode.querySelector("#target", (contentNodeId) => {
+            if (contentNodeId) {
+                targetNode = WI.domManager.nodeForId(contentNodeId);
+                nodeStyles = WI.cssManager.stylesForNode(targetNode);
+                suite.runTestCasesAndFinish();
+            } else {
+                InspectorTest.fail("DOM node not found.");
+                InspectorTest.completeTest();
+            }
+        });
+    });
+}
+</script>
+</head>
+<body onload="runTest()">
+    <p>Testing CSS.forcePseudoState.</p>
+
+    <style>
+#target {
```

Comment:
Nit: Is there a better name for the `target` element given there is a `:target` pseudo-class?